packages/appeals-service-api/.env.example

```diff
@@ -4,4 +4,4 @@
 # Prisma supports the native connection string format for PostgreSQL, MySQL, SQLite, SQL Server, MongoDB and CockroachDB.
 # See the documentation for all the connection string options: https://pris.ly/d/connection-strings
 
-DATABASE_URL="sqlserver://0.0.0.0:1433;database=pins_front_office_development;user=sa;password=DockerDatabaseP@22word!;trustServerCertificate=true"
\ No newline at end of file
+DATABASE_URL="sqlserver://localhost:1433;database=pins_front_office_development;user=sa;password=DockerDatabaseP@22word!;trustServerCertificate=true"
```

Comment:
I needed this for it to work for me.

Reply:
Should have caught this in last PR, this isn't the naming I've used for Azure
Can change here or there
Maybe why dev is down?

https://github.com/Planning-Inspectorate/infrastructure-environments/blob/3134210d37501df66e0b20d4ef64704854093db8/app/components/appeals-app-services/locals.tf#L94

Reply:
Good catch - no the current issue on dev - but it will be!

We specify the env var to use in the schema file (datasource.url). I don't mind much which we use, but we need to align.

Reply:
Going to update this in this PR to align with infra, which has a more specific name.